### DISCUSSION TITLE
Updated a SvelteKit app to TW 3 ~ have to stop and re-start the dev server after saving to see changes

### DISCUSSION THREAD

**Discussion by fourwhitesocks** [tailwindlabs/tailwindcss#6543] [Help]
Hello there; yesterday updated from TW 2.2.19 to TW 3.0.2 In a SvelteKit app everything is working fine after updating except after making changes I have to stop my dev server and then re-run npm run dev to see the changes.  I followed the Docs on updating.  

**Other details:***

- I did change the purge to content in the tailwind.config file
- I searched for any class changes which there were not any
- My node version is:  v14.15.1
- Windows machine newer
- Using SvelteKit which is Version 3.42.6
- Postcss is at 8.4.5

* This is not on GitHub but can answer any questions about it.

 Does there need to be a separate 'watch' script now?

**FROM PACKAGE.JSON:**

``"scripts": {
		"dev": "svelte-kit dev",
		"build": "svelte-kit build",
		"preview": "svelte-kit preview",
		"lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
		"format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. ."
	},
	"devDependencies": {
		"@sveltejs/kit": "next",
		"autoprefixer": "^10.4.0",
		"eslint": "^7.32.0",
		"eslint-config-prettier": "^8.3.0",
		"eslint-plugin-svelte3": "^3.2.1",
		"postcss": "^8.4.5",
		"prettier": "^2.4.1",
		"prettier-plugin-svelte": "^2.4.0",
		"svelte": "^3.42.6",
		"tailwindcss": "^3.0.2"
	},
	"type": "module"`
`

**Comment by yasserlens** [comment]
Not sure if this is the right place to report this - but I'm facing the same issue. Were you able to solve this?

**Comment by fourwhitesocks** [comment]
@yasserlens   I WAS able to solve it; took me a while to find the right link to send you!  Mine was a flippin' spelling mistake in an import :/  Take a look at this tread and make sure you read all of the answers ~ that's what I was missing when I tried to find the right spot for you to look; the date of the part that saved me was on Dec 31 just fyi:   https://github.com/tailwindlabs/tailwindcss/discussions/6754  Good luck!  Amy aka 'fourwhitesocks'  :)

**Reply by yasserlens** [reply]
Thanks so much @fourwhitesocks Amy!
I didn't have any errors - but as I was learning Svelte I installed and uninstalled a few packages so I deleted the node_modules folder and ran npm i to start fresh. That solved it!

**Reply by fourwhitesocks** [reply]
OH boy I've been there and Svelte is changing so much; it will prob happen again ;-), now it seems every time I have to reinstall the NM folder something else breaks lol!!
Best of luck!!

**Reply by yasserlens** [reply]
Ok so this was partially solved it seems. adding breakpoint prefixes like "md:" need a local server restart to work :-/ not sure why yet but I'll keep looking. At least adding classes now works without restarting the server.
